vars/gruntBuild.groovy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(body) {
+    def config = [:]
```

Comment:
This might help http://mrhaki.blogspot.ie/2009/11/groovy-goodness-setting-closures.html